include/z64audio.h

```diff
@@ -0,0 +1,1208 @@
+    /* 0x024 */ f32 volume;
+    /* 0x028 */ f32 muteVolumeScale;
+    /* 0x02C */ f32 fadeVolumeScale;
+    /* 0x030 */ f32 appliedFadeVolume;
+    /* 0x034 */ f32 unk_34;
+    /* 0x038 */ struct SequenceChannel* channels[16];
+    /* 0x078 */ SeqScriptState scriptState;
+    /* 0x094 */ u8* shortNoteVelocityTable;
+    /* 0x098 */ u8* shortNoteGateTimeTable;
+    /* 0x09C */ NotePool notePool;
+    /* 0x0DC */ s32 skipTicks;
+    /* 0x0E0 */ u32 scriptCounter;
+    /* 0x0E4 */ char unk_E4[0x74]; // unused struct members for sequence/sound font dma management, according to sm64 decomp
+    /* 0x158 */ s8 soundScriptIO[8];
+} SequencePlayer; // size = 0x160
+
+typedef struct {
+    /* 0x0 */ u8 releaseRate;
+    /* 0x1 */ u8 sustain;
+    /* 0x4 */ AdsrEnvelope* envelope;
+} AdsrSettings; // size = 0x8
+
+typedef struct {
+    /* 0x00 */ union {
+        struct A {
+            /* 0x00 */ u8 unk_0b80 : 1;
+            /* 0x00 */ u8 hang : 1;
+            /* 0x00 */ u8 decay : 1;
+            /* 0x00 */ u8 release : 1;
+            /* 0x00 */ u8 state : 4;
+        } s;
+        /* 0x00 */ u8 asByte;
+    } action;
+    /* 0x01 */ u8 envIndex;
+    /* 0x02 */ s16 delay;
+    /* 0x04 */ f32 sustain;
+    /* 0x08 */ f32 velocity;
+    /* 0x0C */ f32 fadeOutVel;
+    /* 0x10 */ f32 current;
+    /* 0x14 */ f32 target;
+    /* 0x18 */ char unk_18[4];
+    /* 0x1C */ AdsrEnvelope* envelope;
+} AdsrState; // size = 0x20
+
+typedef struct {
+    /* 0x00 */ u8 unused : 2;
+    /* 0x00 */ u8 bit2 : 2;
+    /* 0x00 */ u8 strongRight : 1;
+    /* 0x00 */ u8 strongLeft : 1;
+    /* 0x00 */ u8 stereoHeadsetEffects : 1;
+    /* 0x00 */ u8 usesHeadsetPanEffects : 1;
+} StereoData;
+
+typedef union {
+    /* 0x00 */ StereoData s;
+    /* 0x00 */ u8 asByte;
+} Stereo; // size = 0x1
+
+typedef struct {
+    /* 0x00 */ u8 reverb;
+    /* 0x01 */ u8 unk_1;
+    /* 0x02 */ u8 pan;
+    /* 0x03 */ u8 unk_3; // Possibly part of stereo?
+    /* 0x04 */ Stereo stereo;
+    /* 0x05 */ u8 unk_4;
+    /* 0x06 */ u16 unk_6;
+    /* 0x08 */ f32 freqScale;
+    /* 0x0C */ f32 velocity;
+    /* 0x10 */ s16* filter;
+    /* 0x14 */ s16* filterBuf;
+} NoteAttributes; // size = 0x18
+
+typedef struct VibratoSubStruct {
+    /* 0x00 */ u16 vibratoRateStart;
+    /* 0x02 */ u16 vibratoExtentStart;
+    /* 0x04 */ u16 vibratoRateTarget;
+    /* 0x06 */ u16 vibratoExtentTarget;
+    /* 0x08 */ u16 vibratoRateChangeDelay;
+    /* 0x0A */ u16 vibratoExtentChangeDelay;
+    /* 0x0C */ u16 vibratoDelay;
+} VibratoSubStruct; // size = 0xE
+
+typedef struct SequenceChannelSubStruct {
+    /* 0x00 */ u8 enabled : 1;
+    /* 0x00 */ u8 finished : 1;
+    /* 0x00 */ u8 stopScript : 1;
+    /* 0x00 */ u8 stopSomething2 : 1; // sets SequenceLayer.stopSomething
+    /* 0x00 */ u8 hasInstrument : 1;
+    /* 0x00 */ u8 stereoHeadsetEffects : 1;
+    /* 0x00 */ u8 largeNotes : 1; // notes specify duration and velocity
+    /* 0x00 */ u8 unused : 1;
+} SequenceChannelSubStruct; // size = 0xE
```

Comment:
The size is wrong here, right?
```suggestion
} SequenceChannelSubStruct; // size = 0x01
```